includes/delete_datastream.form.inc

```diff
@@ -28,9 +31,25 @@ function islandora_delete_datastream_form(array $form, array &$form_state, Abstr
     t('Cancel')
   );
 }
-
 /**
- * Submit handler for the delete datastream form.
+ * Purges datastream derivatives if selected.
```

Comment:
I don't think there is any 'if' here the function just does the job. Perhaps something like `Purge a given DSID's derivative datastreams on an object.` would work.